test/integration/ovs/ofctrl_test.go

```diff
@@ -355,6 +355,51 @@ func TestBundleErrorWhenOVSRestart(t *testing.T) {
 	require.Equal(t, 0, failCount, "No fail case expected")
 }
 
+func TestReconnectOFSwitch(t *testing.T) {
+	br := "br07"
+	err := PrepareOVSBridge(br)
+	require.Nil(t, err, fmt.Sprintf("Failed to prepare OVS bridge: %v", err))
+	defer func() {
+		err = DeleteOVSBridge(br)
+		require.Nil(t, err, fmt.Sprintf("error while deleting OVS bridge: %v", err))
```

Comment:
I don't think this `require` is needed. You should probably not have a `require` in a deferred cleanup function.

Reply:
Sure, I will remove it.